doc/md/guides/orms/sequelize/triggers.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Using Database Triggers in Sequelize
+id: triggers
+slug: triggers
+---
+
+Triggers are useful tools in relational databases that allow you to execute custom code when specific events occur on a
+table. For instance, triggers can automatically populate the audit log table whenever a new mutation is applied to a different table.
+This way we ensure that all changes (including those made by other applications) are meticulously recorded, enabling the enforcement
+on the database-level and reducing the need for additional code in the applications.
+
+This guide explains how to attach triggers to your Sequelize models and configure the schema
+migration to manage both the triggers and the Sequelize models as a single migration unit using Atlas.
+
+:::info [Atlas Pro Feature](/features#pro-plan)
+Atlas support for [Triggers](/atlas-schema/hcl#trigger) used in this guide is available exclusively
+to Pro users. To use this feature, run:
+```
+atlas login
+```
+:::
+
+## Getting started with Atlas and Sequelize
+Before we continue, ensure you have installed the [Atlas Sequelize Provider](https://github.com/ariga/atlas-provider-sequelize)
+on your Sequelize project.
+
+To set up, follow along the [getting started guide](/guides/orms/sequelize) for Sequelize and Atlas.
+
+## Composite Schema
+
+The Sequelize models are mostly used for defining tables and interacting with the database. Triggers
+or any other database native objects do not have representation in Sequelize models.
+
+In order to extend our PostgreSQL schema to include both our Sequelize models and their policies, we configure Atlas to
+read the state of the schema from a [Composite Schema](/atlas-schema/projects#data-source-composite_schema)
+data source. Follow the steps below to configure this for your project:
+
+1\. Let's define a simple model with two types (tables): `users` and `user_audit_logs`:
+
+
+<Tabs>
+<TabItem value={"user"} label={"user.js"}>
+
+```javascript title="user.js"
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+    const User = sequelize.define('User', {
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false
+        }
+    });
+    return User;
+};
+```
+
+</TabItem>
+<TabItem value={"userAuditLog"} label={"userAuditLog.js"}>
+
+```javascript title="userAuditLog.js"
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+    const UserAuditLog = sequelize.define('UserAuditLog', {
+        operationType: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        operationTime: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        newValue: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        oldValue: {
+            type: DataTypes.STRING,
+            allowNull: true
+        },
+    });
+    return UserAuditLog;
+};
```

Comment:
I tested all the triggers (create, delete, update) , and in order to it to work, it need to look like this
```suggestion
'use strict';
module.exports = function(sequelize, DataTypes) {
    const UserAuditLog = sequelize.define('UserAuditLog', {
        operationType: {
            type: DataTypes.STRING,
            allowNull: false
        },
        operationTime: {
            type: DataTypes.STRING,
            allowNull: false
        },
        newValue: {
            type: DataTypes.STRING,
            allowNull: true
        },
        oldValue: {
            type: DataTypes.STRING,
            allowNull: true
        },
    },
    {
         timestamps: false,
    }
    );
    return UserAuditLog;
};
```